cg/services/illumina/backup/backup_service.py

```diff
@@ -278,34 +288,60 @@ def retrieve_archived_file(self, archived_file: Path, run_dir: Path) -> None:
             file_path=str(archived_file), target_path=str(retrieved_file)
         )
 
+    @staticmethod
+    def parse_dsmc_output_sequencing_path(dsmc_output: list[str]) -> list[DsmcSequencingFile]:
+        """Parses the DSMC command output to extract validated sequencing paths."""
+        validated_responses = []
+        for line in dsmc_output:
+            if contains_dsmc_sequencing_path(line):
+                parts = line.split()
+                try:
+                    query_response = DsmcSequencingFile(
+                        date=f"{parts[DsmcOutput.DATE_COLUMN_INDEX]} {parts[DsmcOutput.TIME_COLUMN_INDEX]}",
+                        sequencing_path=parts[DsmcOutput.PATH_COLUMN_INDEX],
+                    )
+                    validated_responses.append(query_response)
+                except ValidationError as e:
+                    LOG.error(f"Validation error for line: {line}\nError: {e}")
```

Comment:
I considered it an error since we can’t tell which file is the newest without a timestamp. I have assumed the timestamp is always correctly formatted. Should I make it more robust in case it’s not? I could ask DSMC if their output is always consistent.

Reply:
Okay, but to me this code says that it is not an error. Excepting an error means that we tell the code "do not exit here, this is fine". So would we want to exit the code here if the validation fails?

Reply:
You are correct, fuuuck feel a bit stupid now..... Let me fix that. I should also make a test for it.

Reply:
Don't feel stupid! The way exception handling is done is not super consistent throughout the code base and sometimes logging the error is the correct course of action. I just wanted to double check if that's the behaviour we are after here 😊